weave/trace_server/validation.py

```diff
@@ -92,3 +93,58 @@ def object_id_validator(s: str) -> str:
 
 def refs_list_validator(s: typing.List[str]) -> typing.List[str]:
     return [validation_util.require_internal_ref_uri(ref) for ref in s]
+
+
+MESSAGE_INVALID_PURGE = "Can only purge by specifying one or more ids"
```

Comment:
I see why we are defining this again, but sort of weird to have two identical constants. I think we should keep this message here, don't allow passing it in, and change the `raise InvalidRequest(MESSAGE_INVALID_PURGE)` message in `token_costs.py` to something more general, like: `"Invalid format for purge, please specify one or more IDs using Mongo...."` etc.

Reply:
The constants are slightly different.
In feedback we add "...feedback ids...", and in costs we add "...costs ids...", to assist in where the call site is, Ill update the constant names and make the messages clearer